encrypt_passwords.rb

```diff
@@ -0,0 +1,7 @@
+require './db/setup'
+require './lib/all'
+
+User.all.each do |u|
+  current_password = u.password
+  u.update! password: Digest::SHA256.hexdigest(current_password) 
+end
```

Comment:
:+1: Thanks for thinking of this. It's actually appropriate to add it as a migration, since it _needs_ to be run against pretty much everybody's machine when they pull this.